### PR TITLE
Uplifting Eiffel bundle to use Eiffel Agen version.

### PR DESCRIPTION
**Pull request by tobiasake**
### Description of the Change
Uplifting EIffel bundle from Eiffel Toulouse to Eiffel Agen version.

Issue link: #14 

### Benefits
New Eiffel Agen version will be used in Eiffel bundle.

### Possible Drawbacks
Eiffel Toulouse will not be supported in Eiffel bundle anymore.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: bundles/Eiffel/seed-data/ei_allevents/subscription.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         {
             "conditions" : [
                 {
-                    "jmespath" : "meta.type=='EiffelArtifactCreatedEvent' && data.gav.groupId=='com.othercompany.library'"
+                    "jmespath" : "meta.type=='EiffelArtifactCreatedEvent' && identity=='pkg:maven/com.othercompany.library/required@required'"
```
Comment (e-backmark-ericsson) [inline bundles/Eiffel/seed-data/ei_allevents/subscription.json]: Shouldn't it be data.identity?

**Comment by e-backmark-ericsson** [comment]
I thought it should be data.identity all over, but at the same time I see that in the old version it was data.gav in some places and only gav in some other places. How come? Does it depend on how the rules file is structured?

File: bundles/Eiffel/seed-data/ei_allevents/subscription.json (hunk 1 above)
Comment (tobiasake) [reply bundles/Eiffel/seed-data/ei_allevents/subscription.json]: yepp, thats true, it should be data.identity according to documentation:
https://github.com/eiffel-community/eiffel/blob/master/eiffel-vocabulary/EiffelArtifactCreatedEvent.md

Will update.

Comment (tobiasake) [reply bundles/Eiffel/seed-data/ei_allevents/subscription.json]: Fixed.

File: bundles/Eiffel/seed-data/ei_artifact/subscription.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
         {
             "conditions" : [
                 {
-                    "jmespath" : "gav.groupId=='com.othercompany.library'"
+                    "jmespath" : "data.identity=='pkg:maven/com.othercompany.library/required@required'"
```
Comment (e-backmark-ericsson) [inline bundles/Eiffel/seed-data/ei_artifact/subscription.json]: gav was not prefixed with 'data.' before, but identity is. Whas it wrong in the old subscription? Or should it not be prefixed now either?

**Comment by tobiasake** [comment]
> I thought it should be data.identity all over, but at the same time I see that in the old version it was data.gav in some places and only gav in some other places. How come? Does it depend on how the rules file is structured?

Jenkins job was triggered by those subscriptions. So Jmepaths maybe matches in both situations. But if only "identity" in Jmepaths, then it maybe can match other identitiy from other parts of the event/aggrObject.

File: bundles/Eiffel/seed-data/ei_artifact/subscription.json (hunk 2 above)
Comment (tobiasake) [reply bundles/Eiffel/seed-data/ei_artifact/subscription.json]: can test with data.identity. Maybe works in both way,
Only "identity" works, tried that yesterday.

Comment (tobiasake) [reply bundles/Eiffel/seed-data/ei_artifact/subscription.json]: Works with "data.identity" in jmepath as well. So both ways works.

Comment (e-backmark-ericsson) [reply bundles/Eiffel/seed-data/ei_artifact/subscription.json]: Ok

**Comment by e-backmark-ericsson** [comment]
Ok. Then it all looks fine to me.

**Comment by tobiasake** [comment]
Tested and works both in Kubernetes and Docker.

**Pull request merged**